src/main/java/com/elixr/poc/rest/response/PurchaseResponse.java

```diff
@@ -0,0 +1,19 @@
+package com.elixr.poc.rest.response;
+
+import lombok.Builder;
+import lombok.Data;
+import org.springframework.data.annotation.Id;
+
+import java.util.UUID;
+@Builder
+/* This Class holds the response attributes for purchase related end points */
```

Comment:
Please move this comment above the @builder annotation

Reply:
UPDATED